[IT] Add `Beatmap/Beatmap collaborations` (#9033)
ppy/osu-wiki: wiki/Beatmap/Beatmap_collaborations/it.md

---
tags:
  - collab
  - collabs
---

# Collaborazioni Beatmap

Le **collaborazioni di beatmap**, o ***collabs***, sono singole [difficoltà](/wiki/Beatmap/Difficulty) o [beatmaps](/wiki/Beatmap) con [difficoltà ospite](/wiki/Beatmap/Guest_difficulty) create da più persone. Sebbene il termine *collab* si riferisca solitamente alle beatmap, indica anche qualsiasi altro media creato da più partecipanti, come [storyboard](/wiki/Storyboard) o canzoni.

## Descrizione

> Le collaborazioni di beatmap di solito hanno la parola "Collab" o più nomi utente attaccati al loro nome della difficoltà o alla [descrizione della beatmap](/wiki/Beatmap/Beatmap_description), anche se non è un requisito per i [criteri di classificazione](/wiki/Ranking_Criteria). È anche comune che i collaboratori più assidui inventino amalgami dei loro nomi utente[^name-example] e li usino in modo coerente per tutta la loro carriera di mapping.

Reviewer: ```suggestion
Le collaborazioni di beatmap di solito hanno la parola "Collab" o più nomi utente concatenati tra loro nel nome della difficoltà o nella [descrizione della beatmap](/wiki/Beatmap/Beatmap_description), nonostante non sia un requisito per i [criteri di classificazione](/wiki/Ranking_Criteria). È anche comune che i collaboratori più assidui inventino amalgami dei loro nomi utente[^name-example] e li usino in modo coerente per tutta la loro carriera di mapping.
```

> Il processo di mapping di solito inizia con l'[host della beatmap](/wiki/Beatmap/Beatmap_host) che divide un modello di beatmap pre-[timed](/wiki/Beatmapping/Timing) in diverse parti, segnandole sulla [timeline](/wiki/Client/Beatmap_editor/Timelines) usando dei segnalibri in base alla struttura musicale della canzone. I segmenti relativamente lunghi, come le strofe, vengono solitamente suddivisi ogni 4 o 8 [battute](/wiki/Music_theory/Measure). Queste parti vengono poi assegnate a ciascun collaboratore secondo una distribuzione concordata, ad esempio per segmenti pari o dispari nel caso di due mappers.

Reviewer: idk about the use of the "timed" word, maybe we can do some brainstorming for this

Reviewer: ```suggestion
Il processo di mapping di solito inizia con l'[host della beatmap](/wiki/Beatmap/Beatmap_host) che suddivide  la beatmap in parti pre-[temporizzate](/wiki/Beatmapping/Timing), segnandole sulla [timeline](/wiki/Client/Beatmap_editor/Timelines) usando dei segnalibri in base alla struttura musicale della canzone. I segmenti relativamente lunghi, come le strofe, vengono solitamente suddivisi ogni 4 o 8 [battute](/wiki/Music_theory/Measure). Queste parti vengono poi assegnate a ciascun collaboratore secondo una distribuzione concordata, ad esempio per segmenti pari o dispari nel caso di due mappers.
```
Not really sure tho... @GYGY88 @marcostudios what do you suggest?

Reviewer 2: usually "temporizzare" is used in different contexts i don't think it really fits here.

```suggestion
Il processo di mapping di solito inizia con l'[host della beatmap](/wiki/Beatmap/Beatmap_host) che suddivide  la beatmap in parti già messe a tempo(/wiki/Beatmapping/Timing), segnandole sulla [timeline](/wiki/Client/Beatmap_editor/Timelines) usando dei segnalibri in base alla struttura musicale della canzone. I segmenti relativamente lunghi, come le strofe, vengono solitamente suddivisi ogni 4 o 8 [battute](/wiki/Music_theory/Measure). Queste parti vengono poi assegnate a ciascun collaboratore secondo una distribuzione concordata, ad esempio per segmenti pari o dispari nel caso di due mappers.
```
this solution may sound a bit too verbose but i guess it works a bit better

Reviewer: ```suggestion
Il processo di mapping di solito inizia con l'[host della beatmap](/wiki/Beatmap/Beatmap_host) che suddivide  la beatmap in parti pre-[timed](/wiki/Beatmapping/Timing), segnandole sulla [timeline](/wiki/Client/Beatmap_editor/Timelines) usando dei segnalibri in base alla struttura musicale della canzone. I segmenti relativamente lunghi, come le strofe, vengono solitamente suddivisi ogni 4 o 8 [battute](/wiki/Music_theory/Measure). Queste parti vengono poi assegnate a ciascun collaboratore secondo una distribuzione concordata, ad esempio per segmenti pari o dispari nel caso di due mappers.
```
Perhaps the `timed` word was not that bad... Maybe it's better since the mappers that make collaborations will have to interact with the international community at some point

> I mapper completano le loro parti disponendo gli [hit objects](/wiki/Gameplay/Hit_object), applicando gli [hitsounds](/wiki/Beatmapping/Hitsound), usando i [timing points](/wiki/Client/Beatmap_editor/Timing#inherited-timing-point) ereditati dove necessario, e poi passano il loro lavoro all'host, che combina le modifiche in un'unica difficoltà modificando accuratamente il suo [`.osu` file](/wiki/Client/File_formats/Osu_(file_format)).

Reviewer: ```suggestion
I mapper completeranno le loro parti disponendo gli [oggetti](/wiki/Gameplay/Hit_object), applicando gli [hitsounds](/wiki/Beatmapping/Hitsound), usando i [punti di timing](/wiki/Client/Beatmap_editor/Timing#inherited-timing-point) ereditati dove necessario. Successivamente, manderanno il loro lavoro all'host, che combinerà le modifiche in un'unica difficoltà modificando accuratamente il suo [`.osu` file](/wiki/Client/File_formats/Osu_(file_format)).
```

> Anche se i mapper di solito si basano sulle parti già completate per rendere le transizioni del gameplay più fluide possibile, esistono le cosiddette *blind collabs* come [Mitose Noriko - Unreal Player](https://osu.ppy.sh/beatmapsets/828563#osu/1735958) hostata da ::{ flag=DE }:: [Mao](https://osu.ppy.sh/users/2204515), in cui i mapper non possono vedere le parti dell'altro finché la beatmap non è terminata. In questi casi, gli oggetti hit sono tipicamente collocati alla fine di ogni sezione per indicare il punto di partenza.

Reviewer: ```suggestion
Anche se i mapper di solito si basano sulle parti già completate per rendere le transizioni del gameplay più fluide possibile, esistono le cosiddette *blind collabs* come [Mitose Noriko - Unreal Player](https://osu.ppy.sh/beatmapsets/828563#osu/1735958) hostata da ::{ flag=DE }:: [Mao](https://osu.ppy.sh/users/2204515), in cui i mapper non possono vedere le parti dell'altro finché la beatmap non è terminata. In questi casi, gli oggetti sono tipicamente collocati alla fine di ogni sezione per indicare il punto di partenza.
```

## Note e riferimenti

> [^name-example]: Per esempio, [masazi - mayday B](https://osu.ppy.sh/beatmapsets/74277#osu/233068), una beatmap di [Vass\_Bass](https://osu.ppy.sh/users/188248), ha una difficoltà in collaborazione con [Reiji-RJ](https://osu.ppy.sh/users/305389) chiamata "Vass-RJ".

Reviewer: ```suggestion
[^name-example]: Ad esempio, [masazi - mayday B](https://osu.ppy.sh/beatmapsets/74277#osu/233068), una beatmap di [Vass\_Bass](https://osu.ppy.sh/users/188248), ha una difficoltà in collaborazione con [Reiji-RJ](https://osu.ppy.sh/users/305389) chiamata "Vass-RJ".
```
